OpenKh.Tests/Patcher/PatcherTests.cs

```diff
@@ -514,6 +514,638 @@ public void MergeKh2AreaDataScriptTest()
+            });
+
+        }
+
+        [Fact]
+        public void ListReplaceBonsTest()
+        {
+            var patcher = new PatcherProcessor();
+            var serializer = new Serializer();
+            var patch = new Metadata()
+            {
+                Assets = new List<AssetFile>()
+                {
+                    new AssetFile()
+                    {
+                        Name = "00battle.bar",
+                        Method = "binarc",
+                        Source = new List<AssetFile>()
+                        {
+                            new AssetFile()
+                            {
+                                Name = "bons",
+                                Method = "listreplace",
+                                Type = "List",
+                                Source = new List<AssetFile>()
+                                {
+                                    new AssetFile()
+                                    {
+                                        Name = "BonsList.yml",
+                                        Type = "bons"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            File.Create(Path.Combine(AssetsInputDir, "00battle.bar")).Using(stream =>
+            {
+                var bonsEntry = new List<Kh2.Battle.Bons>()
+                {
+                    new Kh2.Battle.Bons
+                    {
+                        CharacterId = 1,
+                        RewardId = 15,
+                        BonusItem1 = 10
+                    }
+                    };
+                using var bonsStream = new MemoryStream();
+                Kh2.Battle.Bons.Write(bonsStream, bonsEntry);
+                Bar.Write(stream, new Bar() {
+                    new Bar.Entry()
+                    {
+                        Name = "bons",
+                        Type = Bar.EntryType.List,
+                        Stream = bonsStream
+                    }
+                });
+            });
+
+            File.Create(Path.Combine(ModInputDir, "BonsList.yml")).Using(stream =>
+            {
+                var writer = new StreamWriter(stream);
+                writer.WriteLine("15Sora:");
+                writer.WriteLine("  RewardId: 15");
+                writer.WriteLine("  CharacterId: 1");
+                writer.WriteLine("  BonusItem1: 200");
+                writer.Flush();
+            });
+
+            patcher.Patch(AssetsInputDir, ModOutputDir, patch, ModInputDir);
+
+            AssertFileExists(ModOutputDir, "00battle.bar");
+
+            File.OpenRead(Path.Combine(ModOutputDir, "00battle.bar")).Using(stream =>
+            {
+                var binarc = Bar.Read(stream);
+                var bonsStream = Kh2.Battle.Bons.Read(binarc[0].Stream);
+                Assert.True(bonsStream[0].BonusItem1 == 200);
```

Comment:
Prefer to use `Assert.Equal(200, bonsStream[0].BonusItem1)` instead

Reply:
I assume this is true of the `Assert.True`s in the test suite?

Reply:
Yes